eagle-topology-check/eagle-topology-app/src/main/resources/META-INF/providers/org.apache.eagle.topology.TopologyCheckAppProvider.xml

```diff
@@ -88,14 +88,20 @@
             <description>Hbase Master Principal</description>
             <value>hadoop/_HOST@EXAMPLE.COM</value>
         </property>
-     
+
         <property>
-            <name>dataSourceConfig.hbase.kerberos.master.principal</name>
-            <displayName>Hbase Master Principal</displayName>
+            <name>dataSourceConfig.hbase.kerberos.eagle.principal</name>
```

Comment:
Does it work? I see the code

            hBaseConfig.hbaseMasterPrincipal = config.getString("dataSourceConfig.hbase.kerberos.master.principal");

if (hBaseConfig.eaglePrincipal != null && hBaseConfig.eagleKeytab != null
                && !hBaseConfig.eaglePrincipal.isEmpty() && !hBaseConfig.eagleKeytab.isEmpty()) {
            this.kerberosEnable = true;
            this.hBaseConfiguration.set("hbase.security.authentication", "kerberos");
            this.hBaseConfiguration.set("hbase.master.kerberos.principal", hBaseConfig.hbaseMasterPrincipal);
        }